src/RHOCore.js

```diff
@@ -94,6 +94,18 @@ function toJSData(par /*: IPar */) /*: Json */{
           && Array.isArray(ex.e_list_body.ps)) {
         return ex.e_list_body.ps.map(recur);
       }
+      if (typeof ex.e_map_body !== 'undefined' && ex.e_map_body !== null
+          && Array.isArray(ex.e_map_body.kvs)) {
+        const props = ex.e_map_body.kvs.map((kv) => {
+          const key = recur(kv.key || {});
+          if (typeof key !== 'string') {
+            throw new Error(`not RHOCore? ${JSON.stringify(key)}`);
+          }
+          const val = recur(kv.value || {});
+          return { k: key, v: val };
+        });
+        return props.reduce((acc, { k, v }) => ({ [k]: v, ...acc }), {});
+      }
```

Comment:
Looks like you're adding rholang maps to RHOCore, but I'm not good enough at reading functional-style JS to understand what's happening here.

Reply:
I haven't decided to represent JSON `{ "x": 1 }` as rholang `{"x":1}`
instead of `@#x"!(1)` yet, but when going back the other way, I see
no reason not to support both.

_again, explained in commit message of f2a0ca5 but belongs in a source code comment._